Model/Quote/Checkout.php

```diff
@@ -278,7 +279,18 @@ public function checkPackageType(string $carrier, ?string $country): string
         $this->package->setMailboxActive($this->helper->getBoolConfig($carrierPath[$carrier], 'mailbox/active'));
         $this->package->setWeightFromQuoteProducts($products);
 
-        return $this->package->selectPackageType($products);
+        return $this->package->selectPackageType($products, $carrierPath[$carrier]);
+    }
+
+    /**
+     * @return bool
+     */
+    public function isAgeCheckActive($carrierPath): bool
```

Comment:
What is de input var, een string? Ik ben hier ook in de war, want het heet $carrierPath en je stopt erin (regel 169) $carrierPath[$carrier], dat kan natuurlijk, maar die benaming $carrierPath is dan ambigu geworden voor mij

Reply:
`$carrierPath[$carrier]` is gevuld met `myparcelnl_magento_postnl_settings/`-> dus dit is de daadwerkelijke path die je krijgt voor de carrier PostNL. Op een eerder moment wordt er al door de carriers geloopt.

Reply:
ja maar moet er dan niet string $carrierPath staan? En waarom heet de array waardoor geloopt wordt ook $carrierPath? dat bedoel ik